tests/network/cluster_addons_operator/test_network_addons_operator.py

```diff
@@ -149,21 +160,33 @@ def check_components(network_addons_config_scope_session):
 
 @pytest.fixture(scope="module")
 def net_add_op_bridge_device(worker_node1):
-    with utilities.network.network_device(
-        interface_type=LINUX_BRIDGE,
-        nncp_name="test-network-operator",
-        interface_name="net-add-br",
+    desired_state = DesiredState(
+        interfaces=[
+            Interface(
+                name="net-add-br",
+                state=Resource.Interface.State.UP,
```

Comment:
```suggestion
                state=NetLibsNodeNetworkConfigurationPolicy.Interface.State.UP,
```
Then you can also remove the import of Resource.